MIDIchlorians/MIDIchlorians/TimelineView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  TimelineView.swift
+//  MIDIchlorians
+//
+//  Created by Zhi An Ng on 29/3/17.
+//  Copyright © 2017 nus.cs3217.a0118897. All rights reserved.
+//
+
+import UIKit
+
+class TimelineView: UIView {
+    // a data structure of animation frames
+    // keep track of which frame is selected currently and show it differently
+    private let fullCircle: CGFloat = CGFloat(2 * M_PI)
+    private var selectedFrameIndex: Int?
+    let frames: [Bool] = [
+        true, false,
+        true, false,
+        true, false,
```

Comment:
Boolean values are to indicate if at least one gird has been assigned a colour in this frame?